lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -2793,6 +2793,10 @@ void __cdecl _alloca_probe_16()
             {
                 *value = scriptContext->GetLibrary()->GetNull();
             }
+            else
```

Comment:
In if is it guaranteed that funcCaller->GetScriptContext() is requestContext here?
Better to use nullValue (from line 2754) here.